app/src/components/Loader.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import ReactLoading from "react-loading";
 import styled from "styled-components";
 
-export default ({ size = "3rem" }) => (
+export const Loader = ({ size = "3rem" }) => (
```

Comment:
Même problème qu'au dessus.

```suggestion
export default function Loader({ size = "3rem" }) => {
  return (
```